doc/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.5)
 
 install(FILES
-  pekwm.1
+  pekwm.1.md
```

Comment:
Ahh... Of course, but I think the install should be dropped altogether for now. And either have a md2man (for example go-md2man) as build dependency or check in pre-converted manual pages.

Reply:
Checking in pre-converted manpages would at least prevent that everybody compiling pekwm from source would have to install a tool to convert from markdown to manpage.

I did a quick search and found that we could possibly use ronn, go-md2man or also pandoc. Maybe the most popular one or the one with little dependencies.

Reply:
Yeah, I've been using go-md2man for testing which seems to be working fine. Maybe add some pre-commit hooks for sorting that out on the development side of things